src/hooks/useLiveLogs.ts

```
@@ -0,0 +1,406 @@
+    console.log('agg', agg)
+    
+    // Build message
+    if (count === 1) {
+      // For single events
+      const event = events[0];
+      const eventTypeName = formatEventType(event.type);
+      return `${eventTypeName} of ${formatAmount(event.amount)} USDC`;
+    } else {
+      // For multiple events
+      const typeDescriptions = Object.entries(eventTypes)
+        .map(([type, count]) => `${count} ${formatEventType(type)}${count !== 1 ? 's' : ''}`)
```

ESLint

[error] 'count' is already declared in the upper scope on line 329 column 13.

(@typescript-eslint/no-shadow)